tests/wpt/mozilla/tests/css/body_bgcolor_attribute_change.html

```diff
@@ -0,0 +1,13 @@
+<!doctype html>
+<html>
```

Comment:
nit: The `<html>` and `</html>` tags are redundant and removing them allows you to deindent some code, your call.